src/tests/JIT/opt/Remainder/Regressions/Regression1/Regression1.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <DisableProjectBuild Condition="'$(RuntimeFlavor)' == 'Mono'">true</DisableProjectBuild>
```

Comment:
Why is this disabled on mono?  And if so, why DisableProjectBuild rather than using issues.targets?  (I'm not sure why one would be used over the other except I guess DisableProjectBuild could be used for code that doesn't even build for a platform?)

(And if left this way, is it `Mono` or `mono`?)

Reply:
Disabled this for mono because it was failing the AllSubsets_Mono_LLVMFullAot runs. 

I used disable project build because it was more convenient.

Reply:
As far as using `Mono` or `mono`, I was just following precedent on how it was used. You can see uses from `git grep "DisableProjectBuild Condition="`

Reply:
Did you file a mono issue?  Using issues.targets is the common place to then store the issue number so that the test can later be reenabled.

Grepping for it shows both Mono and mono.

Reply:
We should file the mono issue once the test is checked in.

Reply:
I'd recommend filing it first and then you just include the work item # in a comment (presumably in issues.targets) like other issues are tracked.  Otherwise you'll have a two-step checkin.

Reply:
done